cvc4/include/cvc4_solver.h

```diff
@@ -72,6 +74,17 @@ class CVC4Solver : public AbsSmtSolver
                  const Sort & sort2,
                  const Sort & sort3) const override;
   Sort make_sort(SortKind sk, const SortVec & sorts) const override;
+  Sort make_sort(const DatatypeDecl & d) const override;
+
+  DatatypeDecl make_datatype_decl(const std::string & s) override;
+  DatatypeConstructorDecl make_datatype_constructor_decl(const std::string s) const override;
+  void addConstructor(DatatypeDecl & dt, const DatatypeConstructorDecl & con) const override;
```

Comment:
Let's stick to the underscore naming scheme, so this should be `add_constructor`.